templates/oe-corporate-blocks-site-switcher.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Default theme implementation for rendering corporate site switcher block.
+ *
+ * Available variables:
+ * - links: list of site switcher links.
+ *
+ * @ingroup themeable
+ */
+#}
+
+<div class="site-switcher">
+  <ul>
+    {% for link in links %}
+      <li>
+        <a href="{{ link.href }}" class="{{ link.active ? 'active' }}">{{ link.label }}</a>
```

Comment:
Tiny issue here, we are outputting an empty `class` when the link is not active.